.github/workflows/release-preview.yml

```diff
@@ -0,0 +1,168 @@
+# Matomo release action for automated PREVIEW releases
+#
+# Required GitHub secrets:
+#
+# GPG_CERTIFICATE  |  ASCII armored or Base64 encoded GPG certificate that is used to create the signatures for the archives
+# GPG_CERTIFICATE_PASS  |  Passphrase of the GPG key
+
+name: Build preview release
+
+permissions:
+  actions: read # required for the tests job
+  checks: none
+  contents: write  # required to create tag and release
+  deployments: none
+  issues: read # required for the tests job
+  packages: none
+  pull-requests: read # required for the tests jobs
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+on:
+  # TODO: remove manual dispatch after testing and enable cron
+  workflow_dispatch:
+    branches:
+      - 5.x-dev
+    inputs:
+      password:
+        description: 'Release password'
+        required: true
+  #schedule:
+  #  - cron: '0 1 * * *' # 1am daily
+
+jobs:
+  prepare_preview_version:
+    runs-on: ubuntu-latest
+    outputs:
+      do_release: ${{ steps.changes.outputs.do_release }}
+      has_new_version: ${{ steps.version.outputs.has_new_version }}
+    steps:
+      - name: "Check release password"
+        if: ${{ inputs.is_preview != true && github.event.inputs.password != env.RELEASE_PASSWORD }}
```

Comment:
Are we missing the `env` variable? Or did you mean to use `secrets.RELEASE_PASSWORD`, to avoid having the global environment variable the `release.yml` workflow is using?

Reply:
Keeping it the same as the release workflow so added it back.